src/Octoshift/GithubApi.cs

```diff
@@ -447,12 +447,12 @@ public virtual async Task<string> StartBbsMigration(string migrationSourceId, st
             );
         }
 
-        public virtual async Task<(string State, string RepositoryName, string FailureReason)> GetMigration(string migrationId)
+        public virtual async Task<(string State, string RepositoryName, string FailureReason, string LogUrl)> GetMigration(string migrationId)
```

Comment:
```suggestion
        public virtual async Task<(string State, string RepositoryName, string FailureReason, string MigrationLogUrl)> GetMigration(string migrationId)
```

Reply:
I also have another concern, my understanding is that the migration log url won't be available until the migration is finished (failed or succeeded). If that's the case then in most of `GetMigration()` calls the `MigrationLogUrl` will be `null`. otherwise if it returns a url every single time, b/c we don't use it until the migration is failed or succeeded I'd be more inclined to return the Org name instead and then call the `GetMigrationLogUrl` as needed.

Reply:
Fair point. We only use the url value when the migration succeeds/fails. Is it worth making another call at that point?

Reply:
Well I added a comment suggesting that but then I though if most of the times it's returning `null` then we should be good otherwise it'd worth returning the `OrgName` instead of the `MigrationLogUrl` and then make the call once we need it. Another bonus point would be, by having the `OrgName` we can compose the full log command for `downlod-logs` and show it to the user.

Reply:
True! I'll go ahead and make this change...

Reply:
@ArinGhazarian 

Repo Migration doesn't return the Target Org Name, only the Org Migration API call does that. I'm assuming running the Org Migration API call would return null in this scenario and it doesn't appear that we store the orgname globally. Are my assumptions wrong? Or maybe I'm missing a way to access the org name? 

![0B301551-CFDF-4E00-B648-3A6A00D2F1B5](https://user-images.githubusercontent.com/38845272/219116553-df816fce-881d-43af-86b3-62954e21ae5f.jpeg)

Reply:
OK in that case we don't have any better choice.  I'd just rename the `LogUrl` to `MigrationLogUrl` for clarity in method signature.